docs/book/Makefile

```diff
@@ -31,6 +33,22 @@ RELEASELINK := $(TOOLS_BIN_DIR)/mdbook-releaselink
 $(RELEASELINK): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/mdbook-releaselink ./mdbook/releaselink
 
+MDBOOK := $(TOOLS_BIN_DIR)/mdbook
+$(MDBOOK):
+	vendor/install.sh --git rust-lang/mdBook --tag v0.4.1 --to $(TOOLS_BIN_DIR) --force
+
+MDBOOK_LINKCHECK := $(TOOLS_BIN_DIR)/mdbook-linkcheck
+$(MDBOOK_LINKCHECK):
+	vendor/install.sh --git Michael-F-Bryan/mdbook-linkcheck --tag v0.7.0 --to $(TOOLS_BIN_DIR) --force
```

Comment:
Looks good, let's move the script maybe under `docs/third_party/crate_install.sh`? 

Other than that, 👍